social-providers/ios-profile-gamecenter/SoomlaiOSProfileGameCenter/SoomlaGameCenter.m

```diff
@@ -255,69 +189,70 @@ - (void)getContacts:(bool)fromStart success:(contactsActionSuccess)success fail:
     }];
 }
 
-/**
- Fetches the user's feed
-
- @param success a contacts fetch success callback
- @param fail a contacts fetch failure callback
- */
-- (void)getFeed:(bool)fromStart success:(feedsActionSuccess)success fail:(feedsActionFail)fail {
-    fail(@"GameCenter doesn't support feed.");
-}
-
-
-/**
- Sends an invite
-
- @param success a invite success callback
- @param fail a invite failure callback
- @param cancel a invite cancel callback
- */
-- (void)invite:(NSString *)inviteMessage dialogTitle:(NSString *)dialogTitle success:(inviteSuccess)success
-          fail:(inviteFail)fail cancel:(inviteCancel)cancel {
-    fail(@"GameCenter doesn't support invitations.");
-}
-
-/**
- Shares a photo to the user's feed
-
- @param message A text that will accompany the image
- @param filePath The desired image's location on the device
- @param success an upload image success callback
- @param fail an upload image failure callback
- */
-- (void)uploadImageWithMessage:(NSString *)message
-                   andFilePath:(NSString *)filePath
-                       success:(socialActionSuccess)success
-                          fail:(socialActionFail)fail {
-    fail(@"GameCenter doesn't support image uploading.");
+-(void)getLeaderboards:(BOOL)fromStart success:(successWithArrayHandler)success fail:(failureHandler)fail {
```

Comment:
@EugeneButusov @vedi do we know of such providers where it will be required? If not, I think we can add a method when adding these providers, no need to complicate things just for future use.